src/eduid_userdb/element.py

```diff
@@ -162,6 +162,7 @@ class VerifiedElement(Element):
         is_verified
         verified_by
         verified_ts
+        verification_code
```

Comment:
Not sure all subclasses of Element should have verification_code. E.g. ToUEvent, which inherits from Event -> Element.